api/address.proto

```diff
@@ -9,10 +9,11 @@ message Address {
       TCP = 0;
     }
     Protocol protocol = 1;
-    // For listeners, bind to 0.0.0.0 or :: to support the use of prefix/suffix
-    // matching in FilterChainMatch. For clusters, an address may be either an
-    // IP or hostname to be resolved via DNS.
-    google.protobuf.StringValue address = 2;
+    // For listeners, an empty address implies a bind to 0.0.0.0 or ::. This
```

Comment:
This comment is a little misleading, in the sense that you might bind to a specific address and via iptables rules still make use of the filter stack route matching. I think it's fine to have the default be wildcard, but I would clarify a bit.